crates/core/src/rpc/minimal.rs

```diff
@@ -690,7 +690,7 @@ impl Minimal for SurfpoolMinimalRpc {
         let version = solana_version::Version::default();
 
         Ok(SurfpoolRpcVersionInfo {
-            surfpool_version: format!("surfpool_v{}", SURFPOOL_VERSION),
+            surfnet_version: format!("{}", SURFPOOL_VERSION),
```

Comment:
Hmm, I think of this as surfpool the tool vs surfnet, an instance of the network. So isn't `surfpool_version` what we want?

Regardless, this has downstream implications in txtx - our cheatcode IaC is using this RPC method. If we want this change, can you get a corresponding change there as well?

Reply:
Tracking `surfnet` here seems more adequate. I agree that as it stands today, `surfnet` =  `surfpool` from a version point of view, but this should probably be different in the future, and could help us track breaking changes vs non breaking changes. As in, this version should not be bumped if we're pathing something in the CLI, or in subgraphs, but should be updated when we're tweaking a given RPC. 
This would also help us align local surfnet vs hosted surfnet (not running surfpool directly).

Reply:
Okay, that's fine - can you update the downstream as well? Otherwise, there will be breakages in txtx's `setup_surfnet` action